sormas-backend/src/test/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjbPseudonymizationTest.java

```diff
@@ -87,6 +88,88 @@ public void testEventParticipantOutsideJurisdiction() {
 		assertPseudonymized(getEventParticipantFacade().getEventParticipantByUuid(eventParticipant.getUuid()));
 	}
 
+	@Test
+	public void testPseudonymizedGetByUuidWithLimitedUser() {
+		loginWith(nationalAdmin);
+
+		// event within limited user's jurisdiction
+		EventParticipantDto eventParticipant1 = createEventParticipant(user1, rdcf1);
+		EventDto eventDto1 = getEventFacade().getEventByUuid(eventParticipant1.getEvent().getUuid(), false);
+		eventDto1.setConnectionNumber("123");
```

Comment:
please use the creator method with a consumer so you don't need to update the object right after creation

Reply:
changed the creator method to the one that has a consumer